tests/ops/functions/test_map_wires.py

```diff
@@ -163,6 +163,19 @@ def test_map_wires_nested_tape(self):
         assert nested_m_tape.operations == [qml.PauliY(4), qml.Hadamard(3), qml.PauliY(1)]
         assert len(nested_m_tape.measurements) == 0
 
+    def test_map_wires_batch(self):
+        """Test that map_wires can be applied to a batch of tapes."""
+
+        t1 = qml.tape.QuantumScript([qml.X(0)], [qml.expval(qml.Z(0))])
+        t2 = qml.tape.QuantumScript([qml.Y(1)], [qml.probs(wires=1)])
+
+        batch, _ = qml.map_wires((t1, t2), {0: "a", 1: "b"})
+
+        expected1 = qml.tape.QuantumScript([qml.X("a")], [qml.expval(qml.Z("a"))])
+        expected2 = qml.tape.QuantumScript([qml.Y("b")], [qml.probs(wires="b")])
+        qml.assert_equal(batch[0], expected1)
+        qml.assert_equal(batch[1], expected2)
```

Comment:
Looks like we want support by `assert_equal` of tape batches :sweat_smile:

Reply:
Eventually we'll start getting better error messages for more variety of types.  Kinda hold that in reserve for first time contributors when needed, since it's fairly low stakes and self contained.